wix-members-backend/wix-members-backend/Authentication.service.json

```diff
@@ -1,8 +1,7 @@
 { "name": "Authentication",
   "memberOf": "wix-members-backend",
   "mixes": [],
-  "labels":
-    [ "changed" ],
+  "labels": [],
```

Comment:
Does this needs to be committed ?

Reply:
It's docworks internal stuff.

I will play it safe and do this in the official tool.